src/mon/Monitor.cc

```diff
@@ -2271,6 +2271,110 @@ void Monitor::do_health_to_clog(bool force)
   health_status_cache.summary = summary;
 }
 
+namespace {
+  enum class scrubbed_or_deepscrubbed_t { SCRUBBED, DEEPSCRUBBED };
+
+  void print_unscrubbed_detailed(const std::pair<const pg_t,pg_stat_t> pg_entry,
```

Comment:
I think all of this unscrubbed code should really live here: https://github.com/ceph/ceph/blob/master/src/mon/PGMonitor.cc#L2034